internal/engine/bundle_builder_test.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2023 Stefan Prodan
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package engine
+
+import (
+	"testing"
+
+	"cuelang.org/go/cue/cuecontext"
+	. "github.com/onsi/gomega"
+)
+
+func TestGetBundle(t *testing.T) {
+	g := NewWithT(t)
+	ctx := cuecontext.New()
+
+	t.Run("Get bundle with quoted instance", func(t *testing.T) {
+		bundle := `
+bundle: {
+    apiVersion: "v1alpha1"
+    name:       "podinfo"
+    instances: {
+        "pod-info": {
+            module: url:     "oci://ghcr.io/stefanprodan/modules/podinfo"
+            module: version: "6.3.5"
+            namespace: "podinfo"
+            values: caching: {
+                enabled:  true
+                redisURL: "tcp://redis:6379"
+            }
+        }
+    }
+}
```

Comment:
```suggestion
    instances: {
        "pod-info": {
            module: url:     "oci://ghcr.io/stefanprodan/modules/podinfo"
            module: version: "6.3.5"
            namespace: "podinfo"
            values: caching: {
                enabled:  true
                redisURL: "tcp://redis:6379"
            },
         podinfo: {
            module: url:     "oci://ghcr.io/stefanprodan/modules/podinfo"
            module: version: "6.3.5"
            namespace: "podinfo"
            values: caching: {
                enabled:  true
                redisURL: "tcp://redis:6379"
            }
        }
    }
}
```

Let's add the other variant as well, to test that both are working.

Reply:
this will not work as it is, the test needs adjustments